File: tests/07_phases_ompss_2.dg/fortran/failure_task_reduction_01.f90
! <testinfo>
! test_generator=config/mercurium-ompss-2
! test_compile_fail=yes
! test_nolink=yes
! </testinfo>

subroutine foo()
    implicit none
    integer :: x

    !$oss task weakreduction(+: x) reduction(+: x)
        x = x + 1
    !$oss end task

    !$oss taskwait
end subroutine foo

File: tests/07_phases_ompss_2.dg/c/failure_task_reduction_01.c
/*
<testinfo>
test_generator=config/mercurium-ompss-2
test_compile_fail=yes
test_nolink=yes
</testinfo>
*/

void foo()
{
    int x;

    #pragma oss task weakreduction(+: x) reduction(+: x)
    {
    }

    #pragma oss taskwait
}
